.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cxwangyi/gotorch
+      - image: o0o0o/gotorch
```

Comment:
I push the CI image to my personal repo. Maybe we need a docker organization for GoTorch.

Reply:
Good point. I created the DockerHub account gotorch.